neo/SmartContract/Native/Votes/Method/FindAllPathMethod.cs

```diff
@@ -0,0 +1,209 @@
+﻿using System;
+using System.Collections.Generic;
+
+namespace Neo.SmartContract.Native
+{
+    public class FindAllPathMethod
+    {
+        /// <summary>
+        /// Node Set
+        /// </summary>
+        private List<String> nodesList = new List<String>();
+        /// <summary>
+        /// Vector Graphic,describe the connection between nodes and nodes
+        /// </summary>
+        private Dictionary<String, int> vectorGraphic = new Dictionary<String, int>();
+        /// <summary>
+        /// Edge Array,describe the connection between nodes and nodes
+        /// </summary>
+        private int[,] edgeArray;
```

Comment:
It is an array of connection between edges?
Is it an array or a matrix?

Reply:
![微信图片_20190912113440](https://user-images.githubusercontent.com/42929757/64752232-57959980-d551-11e9-9fbf-caf8053f2fd7.jpg)
It is a matrix descript the between two node and the weight of the path

Reply:
A good idea.So do you have some suggestions?Which variable or function?

Reply:
I will try to push some refactors as soon as possible. Then, you also fell free to suggest any modification or adjustment.

Reply:
@doubiliu, can you proceed with the change? I think it will be easiear.

Let's call it `nodesMatrix` or `nodesWeightsMatrix` or `nodesEdgesMatrix`, something like that for now.

Reply:
nodesEdgesMatrix is better.I will rename it soon.